Libraries/LibWeb/CSS/StyleValues/ImageStyleValue.h

```diff
@@ -46,7 +45,7 @@ class ImageStyleValue final
     virtual Optional<Gfx::Color> color_if_single_pixel_bitmap() const override;
     Gfx::ImmutableBitmap const* current_frame_bitmap(DevicePixelRect const& dest_rect) const;
 
-    Function<void()> on_animate;
+    mutable Function<void()> on_animate;
```

Comment:
This change stood out to me. Why is there enough state in ImageStyleValue that it needs an animation callback? I thought StyleValues were all supposed to be immutable, but this one seems to violate that in a big way 😅

Reply:
ImageStyleValue breaks all kinds of rules and I really don't like it. Suggestions or improvements welcome. 😅